packages/rs-drive-abci/src/execution/platform_events/withdrawals/pool_withdrawals_into_transactions_queue/v1/mod.rs

```diff
@@ -65,8 +79,12 @@ where
                     ))
                 })?;
 
+            // If adding this withdrawal would exceed the limit, stop processing further.
             if potential_total_withdrawal_amount > current_withdrawal_limit {
-                // If adding this withdrawal would exceed the limit, stop processing further.
+                tracing::warn!(
```

Comment:
Not sure it should be warning. Probably just debug because it's normal behaviour. What do you think @QuantumExplorer @lklimek ?

Reply:
Changed to debug